raft/node.go

```diff
@@ -499,6 +502,13 @@ func (n *node) ReportSnapshot(id uint64, status SnapshotStatus) {
 	}
 }
 
+func (n *node) TransferLeadership(ctx context.Context, lead, transferee uint64) {
+	select {
+	case n.recvc <- pb.Message{Type: pb.MsgTransferLeader, From: transferee, To: lead}:
```

Comment:
Yeah I find it strange. But that's how it's implemented now https://github.com/coreos/etcd/blob/master/raft/raft_test.go#L2190-L2213.